ddtrace/appsec/processor.py

```diff
@@ -271,19 +273,30 @@ def on_span_finish(self, span):
                 data[_Addresses.SERVER_REQUEST_BODY] = body
 
         log.debug("[DDAS-001-00] Executing AppSec In-App WAF with parameters: %s", data)
-        res, total_runtime, total_overall_runtime = self._ddwaf.run(data, self._waf_timeout)  # res is a serialized json
+        blocked_request = _context.get_item("http.request.blocked", span=span)
```

Comment:
is this request blocking new?

Reply:
Mmmm odd, this is in the IP Blocking branch (and user id blocking, and suspicious request blocking...), that branches from this one, but should not be on this one specifically.